docs/UserGuide.adoc

```diff
@@ -53,15 +53,15 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 === Viewing help : `help`, `h`, `assist` or `faq`
 
-Format: `help` or `h` +
```

Comment:
Pls merge doc updates to GeneralDocumentation for staging first pls. Our branches are already very messy!!!!!!